test/servico/cst10Test.java

```diff
@@ -53,10 +44,13 @@ public static void tearDownClass() {
     public void setUp() throws ParseException, NonexistentEntityException {
         //Setup Declaração de Ausência Professor 2
         ProfessorService professorService = new ProfessorService();
-        ProfessorModel professor = professorService.obterProfessorPorNome("Professor2");
+        ProfessorModel professor =
+                professorService.obterProfessorPorNome("Professor2");
         
         NotificacaoService notificaçãoService = new NotificacaoService();
-        notificaçãoService.notificarAusencia(professor.id, "25/11/2013 18:30", "25/11/2013 19:59", "Motivo Declarado", new LinkedList<String>());
+        notificaçãoService.notificarAusencia(

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['method def' child have incorrect indentation level 8, expected level should be 4.](https://www.codacy.com/app/calebepb/substituicao/file/2777960760/issues/source?bid=3364462&fileBranchId=3364484#l51)